src/index.html

```diff
@@ -3,11 +3,330 @@
+                >
+              </div>
+              <div class="category__photo">
+                <img
+                  src="./images/photos/img-2.png"
+                  alt="Headphones"
+                  class="category__image"
+                >
+              </div>
+            </div>
+
+            <h5 class="category__name">Headphones & earbuds</h5>
+          </div>
+
+          <div class="category__details">
+            <div class="category__photos">
+              <div class="category__photo">
+                <img
+                  src="./images/photos/img-3.png"
+                  alt="Grey Speaker"
+                  class="category__image"
+                >
+              </div>
+
+              <div class="category__photo">
+                <img
+                  src="./images/photos/img-4.png"
+                  alt="Blue speaker"
+                  class="category__image"
+                >
+              </div>
+            </div>
+
+            <h5 class="category__name">Speakers</h5>
+          </div>
+
+          <div class="category__details">
+            <div class="category__photos">
+              <div class="category__photo">
+                <img
+                  src="./images/photos/img-5.png"
+                  alt="blue audio sunglasses"
+                  class="category__image"
+                >
+              </div>
+
+              <div class="category__photo">
+                <img
+                  src="./images/photos/img-6.png"
+                  alt="Black audio sunglasses"
+                  class="category__image"
+                >
+              </div>
+            </div>
+
+            <h5 class="category__name">Audio sunglasses</h5>
+          </div>
+        </div>
+
+        <a href="#category" class="btn">View all</a>
+      </article>
+
+      <article class="about">
+        <div class="about__wraper">
+          <h2 class="title about__title">
+            Why buy direct
+            <br>
+            from Bose
+          </h2>
+
+          <div class="about__info">
+            <p class="about__benefits">Free 2-day shipping and returns</p>
+            <p class="about__benefits">90-day risk-free trial</p>
+            <p class="about__benefits">World class customer service</p>
+            <p class="about__benefits">My Bose account management</p>
+
+            <p class="about__summary">A great product is more than what&lsquo;s in the box.
+              It&lsquo;s a promise of premium performance, world-class support,
+              and everything you expect from a trusted brand.
+              It&lsquo;s just one of many reasons why you&lsquo;ll
+              shop with confidence on Bose.com.
+            </p>
+          </div>
+        </div>
+      </article>
+
+      <article class="contact-us">
+        <h2 class="title contact-us__title">
+          Contact us
+        </h2>
+
+        <div class="contact-us__wraper">
+          <form
+            action="/"
```

Comment:
if you check the deployed version, submit drops an error
try this
```suggestion
            action="#"
```

Reply:
corrected thank you